### DISCUSSION TITLE
Panic button kills audio output

### DISCUSSION THREAD

**Discussion by William-Purvis** [GrandOrgue/grandorgue#1453] [Help] [answered]
I'm still a novice user of GrandOrgue, currently running GO 3.10.0.0 under Ubuntu 22.04.2.

I got in a mess and had a load of notes stuck on, and hit the panic button. All sound stopped.

However, I couldn't get any further notes to play without restarting!
It seems GO breaks the Jack connection to system output.
If I reconnect it's OK.

Is this supposed to happen?

**Comment by ClarkeBrunt** [comment, marked as answer]
I don't know about 'supposed to happen', but I think it will happen. I believe Panic turns off any notes, and resets the audio. I'm running GO 3.10.1-1 on Ubuntu Studio 22.04.2 LTS, though I doubt that the precise versions are relevant to this. I use Jack native output from GO, but only because I want to send the audio through a graphic equaliser running in Carla. I start Jack with QjackCtl, loading a pre-saved 'patchbay' with the Jack connections I want. This automatically makes Jack connections when the items to be connected appear. I can see in the Messages window in QjackCtl that pressing Panic in GO causes it to disconnect and re-connect to Jack. Because of the patchbay, my connections are re-made automatically, but if you're connecting things manually, then you'll have to do it again.

If it wasn't for wanting the graphic equaliser (and possibly better cooperation with other audio components) then I wouldn't use Jack at all. GO was previously working fine with direct output to the audio device.
